docs/content/direct/packaging.md

```diff
@@ -227,7 +227,7 @@ By our development practices and not doing any manual hacking we maintain the as
 
 The [release process](release.md) builds and publishes that container image.
```

Comment:
This file should also be updated to talk about the newly introduced image. Does not have to be done in this PR.

Reply:
@MikeSpreitzer right. atm we have no new image. the newly introduced image will be done in a followup PR that adds helm chart. this file will be updated accordingly in the same PR.